modules/ml/src/test/java/org/apache/ignite/ml/dataset/impl/local/LocalDatasetBuilderTest.java

```diff
@@ -105,6 +107,7 @@ public void testBuildWithPredicate() {
         };
 
         return builder.build(
+            TestUtils.testEnvBuilder(),
```

Comment:
builder.withEnvBuilder? why not?

Reply:
Yeah, I thought about it too, but decided to go along with pattern with `contextBuilder` (we in majority of cases have EmptyContext() as context, but do not have defaults + `with` for it, and if we opted to change this sad situation, we'd have maybe too much methods in interface). But maybe we should add ticket for this API change:)